perun-core/src/main/resources/postgresChangelog.txt

```diff
@@ -6,6 +6,17 @@
 -- Directly under version number should be version commands. They will be executed in the order they are written here.
 -- Comments are prefixed with -- and can be written only between version blocks, that means not in the lines with commands. They have to be at the start of the line.
 
+3.1.66
+CREATE TABLE vos_bans (id integer not null, member_id integer not null, vo_id integer not null, description varchar, banned_to timestamp not null, created_at timestamp default statement_timestamp() not null, created_by varchar default user not null, modified_at timestamp default statement_timestamp() not null, modified_by varchar default user not null, created_by_uid integer, modified_by_uid integer, constraint vos_bans_pk primary key (id), constraint vos_bans_u unique (member_id), constraint vos_bans_mem_fk foreign key (member_id) references members (id), constraint vos_bans_vo_fk foreign key (vo_id) references vos (id));
+CREATE SEQUENCE "vos_bans_id_seq";
+CREATE INDEX idx_fk_vos_ban_member ON vos_bans (member_id);
+CREATE INDEX idx_fk_vos_ban_vos ON vos_bans (vo_id);
+GRANT ALL ON vos_bans TO perun;
+INSERT INTO vos_bans (id, description, banned_to, member_id, vo_id) SELECT nextval('vos_bans_id_seq'), 'AUTO CREATED FROM SUSPEND STATUS', members.suspended_to, members.id, members.vo_id FROM members WHERE status = 2;
```

Comment:
What about records where suspended_to is null? Do you have a default in the new table? In code you are working with date in the far future.

Reply:
That's a good point. I have added a default value for the table.